src/main/kotlin/app/revanced/patches/gradle/PatchesPlugin.kt

```diff
@@ -241,4 +257,23 @@ private fun Project.configureJarTask(patchesExtension: PatchesExtension) {
             attributes["License"] = patchesExtension.about.license
         }
     }
+
+    val shade = configurations.create("shade")
+    configurations.getByName("compileClasspath").extendsFrom(shade)
+    configurations.getByName("runtimeClasspath").extendsFrom(shade)
+
+    tasks.withType(ShadowJar::class.java).configureEach {
+        it.configurations = listOf(shade)
+        it.archiveClassifier.set("")
+
+        it.minimize()
+        it.isEnableRelocation = true
+        it.relocationPrefix = "app.revanced.libs"
```

Comment:
```suggestion
        it.relocationPrefix = "app.revanced"
```

This should be less arbitrary